conans/client/tools/scm.py

```diff
@@ -120,3 +125,79 @@ def get_revision(self):
             return commit
         except Exception as e:
             raise ConanException("Unable to get git commit from %s\n%s" % (self.folder, str(e)))
+
+
+class SVN(SCMBase):
+    cmd_command = "svn"
+
+    def __init__(self, runner=None, *args, **kwargs):
+        def runner_no_strip(command):
+            return subprocess.check_output(command, shell=True).decode()
+        runner = runner or runner_no_strip
+        super(SVN, self).__init__(runner=runner, *args, **kwargs)
+
+    def clone(self, url, branch=None, submodule=None):
+        assert branch is None, "This concept has no meaning for SVN"
```

Comment:
Doesn't it? As far as I remember there were branches in svn. Could we check out first the repo and then change to the branch?

Reply:
You are right, now I remember. Thanks! it makes sense, the URL will define the "branch" and the revision (integer) will be unique.